dt-contacts/base-setup.php

```diff
@@ -521,6 +529,18 @@ public static function dt_user_list_filters( $filters, $post_type ){
                     [ "id" => 'recent', 'name' => __( "Last 30 viewed", 'disciple_tools' ) ]
                 ]
             ];
+            $filters["filters"][] = [
```

Comment:
I think this filter should be available on all post types by default.
Maybe: `dt-posts/custom-post-type.php`

Reply:
I tried to add this in the `dt-posts/custom-post-type.php` but because the tabs have different names between post types it is hard to do it in a generic way. I have included it in groups as well as contacts, but right now we are doing filters for each post type separately.

Reply:
I'll take a look at this one. I like not having to copy paste the same code for each post type.
We need something similar for the "recently viewed" filter too.